src/plugin/fieldViews/TextFieldView.ts

```diff
@@ -137,18 +149,26 @@ export class TextFieldView extends ProseMirrorFieldView {
           this.innerEditorView.dom
         );
 
+        const domElement = this.innerEditorView.dom as HTMLDivElement;
         if (enableMultiline) {
-          (this.innerEditorView.dom as HTMLDivElement).style.minHeight = `${
+          const initialInputHeightPx = `${
             parseInt(lineHeight, 10) * rows + parseInt(paddingTop) * 2
           }px`;
+          domElement.style.minHeight = initialInputHeightPx;
+          if (isResizeable) {
+            // If the input is resizeable, assume that the user would like the input
+            // to begin life with its height set to `rows`, with the opportunity to
+            // expand it later.
+            domElement.style.height = initialInputHeightPx;
+          }
         }
 
         if (maxRows) {
-          (this.innerEditorView.dom as HTMLDivElement).style.maxHeight = `${
+          const maxHeightPx = `${
             parseInt(lineHeight, 10) * maxRows + parseInt(paddingTop)
           }px`;
-          (this.innerEditorView.dom as HTMLDivElement).style.overflowY =
-            "scroll";
+          domElement.style.maxHeight = maxHeightPx;
+          domElement.style.overflowY = "scroll";
```

Comment:
Great simple implementation with HTML 5, as not aware of this and expected some fancy js!